.env.sample

```diff
@@ -6,4 +6,6 @@ LINKEDIN_CLIENT_SECRET=
 LINKEDIN_REDIRECT_URI=http://localhost:3000/auth/linkedin/callback
 FB_APP_ID = 
 FB_APP_VERSION = 
-FB_REDIRECT_URI=http://localhost:3000/auth/facebook/callback
\ No newline at end of file
+FB_REDIRECT_URI = http://localhost:3000/auth/facebook/callback
+IG_APP_ID = 
```

Comment:
It would be very helpful if you could update [README.md](./README.md) as well so that we know how/where to get these information. If there's a way for you to add us as developers in Instagram as well that would be lovely.